### ISSUE TITLE
How many tokens should be trained when pretraining the text-to-image generation in the paper?

### ISSUE THREAD

**Issue by runzeer** [THUDM/CogView2#30]

**Comment by Sleepychord** [comment]
Do you mean the max-sequence-length? we use 512 length (400 image tokens + up to 112 text tokens). I think the other hyperparams can be found in the paper, except for learning rate, which I need to check the codes after work, but I think any values around 1e-4 with warmup is okay.

**Comment by runzeer** [comment]
I mean the training hyperparameters such like training steps, the training tokens per batch. In the CogView paper, it gave us the relevant parameters.
![1](https://user-images.githubusercontent.com/14292495/182738519-053a5446-2a95-4a6c-9bdf-6bcf01c6c2f6.png)

**Comment by Sleepychord** [comment]
Hi, please refer to section 3.2 of CogView2 paper~

**Comment by runzeer** [comment]
Thanks a lot! Also, I had a question.  when the model can generate the visible image, the loss could be what level. I mean could you give me some training logs so that I can do the experiments as a reference?
